process/coordinator/blockDataRequest_test.go

```diff
@@ -961,6 +929,205 @@ func TestBlockDataRequest_receivedMiniBlock(t *testing.T) {
 	})
 }
 
+func TestBlockDataRequest_ConcurrentOperations(t *testing.T) {
```

Comment:
this is also good, but here i was referring to multiple concurrent calls for all methods in one test, as we have in other components